code/datums/station_traits/negative_traits.dm

```diff
@@ -17,6 +17,30 @@
 /datum/station_trait/distant_supply_lines/on_round_start()
 	SSeconomy.pack_price_modifier *= 1.2
 
+///A negative trait that stops mail from arriving (or the inverse if on holiday). It also enables a specific shuttle loan situation.
+/datum/station_trait/mail_blocked
+	name = "Postal workers strike"
+	trait_type = STATION_TRAIT_NEGATIVE
```

Comment:
I'd be willing to say we swap this over to STATION_TRAIT_NEUTRAL if only because it seems to go either way depending on if it's a holiday or not.

Reply:
6 times out of 7 it's going to be a negative thing, so it'd be unfair to call this neutral, and it'd take several additional steps to avoid this from rolling on holidays and the such since the Station subsystem is loaded before both SSeconomy and SSevents, hence I came up with this device.